src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -26,7 +27,8 @@ class DacFxService
         private static readonly Lazy<DacFxService> instance = new Lazy<DacFxService>(() => new DacFxService());
         private readonly Lazy<ConcurrentDictionary<string, DacFxOperation>> operations =
             new Lazy<ConcurrentDictionary<string, DacFxOperation>>(() => new ConcurrentDictionary<string, DacFxOperation>());
-        private Lazy<ConcurrentDictionary<string, TSqlModel>> modelResults =
+        // Key for a projectModel is the project uri
```

Comment:
For stuff like this it's preferable to have it be an actual xml doc comment with a description of what it is so it shows up in hover text